eng/Helix.SingleTarget.targets

```diff
@@ -15,8 +15,12 @@
     <!-- Set TEST_DOTNET_ROOT to same as DOTNET_ROOT (which is provided by the dotnet cli correlation payload) -->
     <HelixWorkItemPreCommand Include="export TEST_DOTNET_ROOT=${DOTNET_ROOT}"
                              Condition="'$(IsHelixPosixShell)' == 'true'" />
+    <HelixWorkItemPreCommand Include="export LANG=en_US.UTF-8"
```

Comment:
Add a comment referring to the issue that we logged about needing to fix this value and make this only apply when it's targeting the Linux machines e.g. adding `$([MSBuild]::IsOSPlatform(Linux))` to the condition is sufficient.